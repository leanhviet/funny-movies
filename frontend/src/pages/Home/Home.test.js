// Libs
import React from 'react'
import { shallow } from 'enzyme'

// Components
import { Home } from './index'

describe('Home component', () => {
  const wrapper = shallow(<Home />)

  it('should be defined Home component', () => {
    expect(wrapper).toBeDefined()
  })

  // Testing Snapshot
  it('should be render snapshot Home component', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
