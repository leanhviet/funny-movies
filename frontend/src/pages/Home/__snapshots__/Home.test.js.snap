// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should be render snapshot Home component 1`] = `
<Fragment>
  <Memo(HeaderComponent) />
  <Container
    fluid={false}
  />
</Fragment>
`;
